gcp-repo/metadata.p2.inf

```diff
@@ -27,7 +27,7 @@ units.1.properties.5.name=org.eclipse.equinox.p2.doc.url
 units.1.properties.5.value=https://cloud.google.com/eclipse/docs/
 units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
 units.1.provides.1.name=com.google.cloud.tools.eclipse.category
-units.1.provides.1.version=1.0.0
+units.1.provides.1.version=1.0.1
```

Comment:
I think you need to change the `requires.1.range` up above.  This .p2.inf format hurts to read.  Basically this file associates p2 metadata to the `com.google.cloud.tools.eclipse.dist` product defined in the `gcp-repo/metadata.product` file.  
  - the initial `requires.1` adds a requirement on the product IU for an IU with certain capabilities; think of the _namespace_ as being a the capability _type_, and the _name_ is the capability identifier.  So this `requires.1` says the product requires our new category
  - the `units.N` defines new IUs; the `units.N.provides.M` describes the capabilities of this IU. In this case this IU provides a category definition.  The category values are in the properties.
  - the category IU also has some requirements to pull in a feature